inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/popupComponent.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import * as ReactDOM from 'react-dom';
+import { Inspector } from '../../../../../../inspector';
+
+interface IPopupComponentProps {

```

Comment:
we already have a popup component:
https://github.com/BabylonJS/Babylon.js/blob/master/inspector/src/components/actionTabs/tabs/propertyGrids/animations/popupComponent.tsx

could you move this one up so we can share it?

Reply:
Would it be okay to move the popupComponent up to /inspector/src/components? I don't want to clutter that folder but it does seem like an inspector-level component

Reply:
even better :D